server/views/pages/bookingJourney/selectVisitors.njk

```diff
@@ -36,24 +36,24 @@
         {% set visitorList = [] %}
         {% for visitor in visitors %}
           {% set visitorList = (visitorList.push({
-            value: visitor.displayId,
+            value: visitor.visitorDisplayId,
             text: visitor.firstName + " " + visitor.lastName + " (" + visitor.dateOfBirth | displayAge + ")"
           }), visitorList)%}
         {% endfor %}
         <form action="/book-a-visit/select-visitors" method="POST" novalidate>
           <input type="hidden" name="_csrf" value="{{ csrfToken }}">
 
           {{ govukCheckboxes({
-            name: "visitorIds",
+            name: "visitorDisplayIds",
```

Comment:
Be explicit about what property this form field is handling (i.e. it's the computed `visitorDisplayId`, **not** the `visitorId`).